java/java.lsp.server/vscode/src/extension.ts

```diff
@@ -66,7 +67,7 @@ export function enableConsoleLog() {
 }
 
 export function findClusters(myPath : string): string[] {
```

Comment:
This is an exported function, I'd prefer to keep it pure (e.g. without side effects).

Reply:
Fixed.